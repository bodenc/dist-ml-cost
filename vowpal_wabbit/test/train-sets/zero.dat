0 1.0  |txt 0 1 2 3 4 5 57 58 59 60 61 62 114 115 116 117 118 120 171 172 173 174 178 180 232 233 234 235 239 240 293 294 295 296 300 302 346 347 348 349 353 355 399 400 401 402 406 407 454 455 456 464 468 470 509 510 511 512 513 515 562 563 564 572 573 575 621 622 623 624 625 627 672 673 674 675 679 681 729 730 731 739 740 741 784 785 786 787 788 790 837 838 839 840 841 842 884 885 886 887 891 893 945 946 956 957 961 963 1000 1001 1015 1023 1027 1028 1059 1060 1061 1069 1073 1075 1114 1115 1131 1132 1136 1138 1175 1176 1192 1193 1197 1199 1238 1239 1240 1248 1252 1253 1301 1302 1318 1326 1327 1329 1364 1365 1366 1374 1375 1376 1425 1426 1427 1428 1429 1431 1480 1481 1493 1501 1505 1507 1539 1540 1554 1562 1563 1565 1598 1599 1600 1604 1608 1610 1647 1648 1649 1650 1651 1653 1706 1707 1708 1716 1717 1719 1763 1764 1765 1766 1767 1769 1822 1823 1824 1825 1826 1827 1869 1870 1886 1887 1891 1893 1930 1931 1932 1940 1944 1946 1985 1986 1987 1995 1999 2001 2040 2041 2057 2065 2066 2067 2093 2123 2135 2136 2140 2142 2150 2151 2167 2175 2176 2177 2207 2208 2209 2217 2221 2222 2266 2267 2268 2276 2277 2278 2325 2339 2340 2348 2352 2354 2368 2394 2410 2411 2412 2414 2421 2422 2423 2431 2432 2433 2480 2481 2482 2490 2494 2496 2537 2538 2539 2540 2541 2543 2574 2592 2608 2609 2613 2615 2623 2647 2648 2649 2653 2655 2678 2679 2680 2688 2692 2694 2731 2732 2748 2749 2753 2755 2790 2806 2807 2815 2816 2818 2825 2826 2840 2841 2845 2846 2878 2879 2895 2896 2900 2901 2937 2938 2939 2947 2951 2952 2986 3010 3011 3019 3020 3022 3041 3042 3043 3051 3055 3056 3086 3087 3088 3096 3100 3102 3143 3144 3150 3151 3155 3157 3190 3206 3216 3217 3218 3220 3229 3230 3231 3232 3233 3235 3266 3292 3298 3299 3300 3302 3313 3343 3344 3345 3349 3350 3364 3365 3375 3383 3384 3385 3417 3437 3438 3446 3450 3451 3464 3465 3466 3467 3468 3470 3499 3500 3516 3522 3526 3528 3560 3561 3573 3574 3575 3576 3611 3621 3622 3623 3624 3625 3652 3653 3654 3655 3659 3661 3685 3686 3687 3695 3696 3698 3734 3735 3736 3737 3738 3740 3787 3788 3789 3790 3794 3796 3844 3866 3867 3875 3879 3881 3891 3892 3893 3901 3903 3946 3966 3967 3975 3979 3980 3993 4023 4037 4038 4042 4044 4048 4049 4065 4073 4077 4078 4105 4106 4122 4128 4129 4131 4164 4165 4166 4167 4168 4170 4219 4220 4221 4222 4223 4224 4266 4288 4289 4297 4298 4299 4317 4318 4319 4320 4321 4323 4368 4369 4383 4389 4393 4395 4415 4443 4444 4452 4453 4455 4472 4502 4503 4511 4515 4516 4525 4526 4527 4528 4529 4530 4586 4594 4595 4603 4607 4609 4619 4645 4661 4669 4670 4672 4676 4677 4678 4686 4687 4689 4713 4714 4715 4716 4717 4718 4740 4741 4742 4748 4752 4754 4799 4800 4814 4820 4821 4823 4852 4853 4869 4870 4871 4873 4909 4910 4911 4912 4916 4918 4962 4963 4964 4972 4976 5011 5012 5013 5014 5015 5017 5066 5074 5080 5088 5089 5091 5095 5096 5108 5116 5117 5118 5132 5152 5153 5161 5165 5166 5183 5184 5185 5193 5194 5195 
0 1.0  |txt 0 28 42 43 47 49 57 85 99 100 104 105 114 142 156 157 161 163 171 201 202 203 204 205 232 262 278 279 283 284 293 321 333 339 340 342 346 374 375 376 377 379 399 427 428 429 433 434 454 478 479 480 481 483 509 537 538 539 540 542 562 590 591 592 593 594 621 647 648 649 650 651 672 704 716 722 726 727 729 757 758 759 763 765 784 810 811 819 823 824 837 859 860 861 862 863 884 916 917 925 926 927 945 946 947 948 949 950 1000 1001 1002 1003 1004 1006 1059 1091 1092 1093 1094 1095 1114 1115 1116 1117 1118 1119 1175 1176 1177 1178 1179 1180 1238 1270 1271 1272 1276 1277 1301 1333 1334 1335 1339 1340 1364 1394 1395 1396 1400 1402 1425 1426 1427 1428 1429 1431 1480 1508 1509 1510 1514 1515 1539 1540 1541 1547 1548 1549 1598 1616 1617 1618 1622 1624 1647 1648 1649 1650 1651 1653 1706 1707 1708 1716 1717 1719 1763 1764 1780 1781 1782 1783 1822 1823 1824 1825 1826 1827 1869 1870 1871 1872 1873 1875 1930 1931 1932 1940 1944 1946 1985 1986 1987 1995 1999 2001 2040 2041 2057 2065 2069 2071 2093 2123 2135 2136 2140 2142 2150 2151 2167 2175 2176 2177 2207 2208 2224 2225 2226 2228 2266 2267 2268 2276 2277 2278 2325 2339 2340 2348 2352 2354 2368 2369 2383 2387 2388 2390 2421 2422 2423 2424 2425 2427 2480 2481 2497 2498 2499 2500 2537 2538 2539 2540 2541 2543 2574 2592 2608 2609 2613 2615 2623 2647 2648 2649 2653 2655 2678 2679 2680 2688 2692 2694 2731 2732 2748 2756 2760 2762 2790 2806 2807 2815 2816 2818 2825 2826 2840 2841 2845 2846 2878 2879 2895 2896 2900 2901 2937 2938 2939 2940 2941 2942 2986 3010 3011 3019 3020 3022 3041 3042 3043 3044 3048 3050 3086 3087 3088 3096 3100 3102 3143 3144 3150 3151 3155 3157 3190 3206 3216 3217 3218 3220 3229 3230 3231 3232 3233 3235 3266 3292 3298 3299 3300 3302 3313 3314 3328 3336 3340 3341 3364 3365 3375 3383 3384 3385 3417 3437 3438 3446 3450 3451 3464 3465 3466 3467 3468 3470 3499 3500 3516 3522 3526 3528 3560 3561 3573 3581 3582 3584 3611 3621 3622 3623 3624 3625 3652 3653 3654 3655 3659 3661 3685 3686 3687 3695 3696 3698 3734 3735 3736 3744 3745 3747 3787 3788 3789 3790 3791 3792 3844 3866 3867 3875 3879 3881 3891 3892 3893 3894 3898 3899 3946 3966 3967 3975 3979 3980 3993 4023 4037 4038 4042 4044 4048 4049 4065 4073 4077 4079 4105 4106 4122 4128 4129 4131 4164 4165 4166 4167 4168 4170 4219 4220 4221 4222 4223 4224 4266 4288 4289 4290 4294 4295 4317 4318 4319 4320 4324 4325 4368 4369 4383 4389 4393 4395 4415 4443 4444 4452 4456 4458 4472 4502 4503 4511 4512 4513 4525 4526 4527 4535 4536 4537 4586 4594 4595 4603 4607 4609 4619 4645 4661 4669 4670 4672 4676 4677 4678 4686 4687 4689 4713 4714 4715 4716 4717 4718 4740 4741 4742 4748 4752 4754 4799 4800 4814 4820 4821 4823 4852 4853 4869 4870 4874 4875 4909 4910 4911 4912 4916 4918 4962 4963 4964 4965 4966 4967 5011 5012 5013 5014 5015 5017 5066 5074 5080 5088 5089 5091 5095 5096 5108 5109 5110 5111 5132 5152 5153 5161 5165 5166 5183 5184 5200 5208 5209 5210 
0 1.0  |txt 0 1 2 10 11 12 57 58 59 67 68 69 114 115 116 124 125 126 171 172 173 181 182 184 232 233 234 235 239 240 293 294 295 303 304 306 346 347 348 349 353 355 399 400 401 402 406 407 454 455 456 457 461 462 509 510 511 512 513 515 562 563 564 572 573 575 621 622 623 631 632 634 672 673 674 675 676 677 729 730 731 732 736 737 784 785 786 787 788 790 837 838 839 840 841 842 884 885 886 894 898 899 945 946 947 948 949 950 1000 1001 1002 1003 1004 1006 1059 1060 1061 1069 1073 1075 1114 1115 1116 1117 1118 1119 1175 1176 1177 1178 1179 1180 1238 1270 1286 1287 1291 1292 1301 1333 1349 1350 1354 1355 1364 1394 1410 1411 1415 1416 1425 1426 1427 1428 1429 1431 1480 1508 1524 1525 1526 1527 1539 1540 1541 1547 1548 1549 1598 1616 1632 1633 1637 1638 1647 1648 1649 1650 1651 1653 1706 1738 1754 1755 1759 1760 1763 1764 1765 1766 1770 1771 1822 1846 1860 1861 1862 1864 1869 1870 1871 1872 1873 1875 1930 1931 1932 1940 1944 1946 1985 2017 2031 2032 2033 2034 2040 2041 2042 2043 2047 2048 2093 2123 2135 2136 2140 2142 2150 2182 2198 2199 2200 2202 2207 2208 2209 2210 2211 2212 2266 2267 2268 2276 2277 2278 2325 2339 2355 2363 2364 2365 2368 2369 2383 2387 2388 2390 2421 2422 2423 2424 2425 2427 2480 2481 2482 2483 2484 2485 2537 2538 2539 2540 2541 2543 2574 2592 2608 2609 2613 2615 2623 2647 2648 2649 2653 2655 2678 2704 2705 2706 2707 2708 2731 2732 2733 2734 2738 2739 2790 2806 2807 2815 2816 2818 2825 2826 2840 2841 2845 2846 2878 2910 2926 2930 2931 2932 2937 2938 2939 2940 2941 2942 2986 3010 3011 3019 3020 3022 3041 3042 3043 3044 3048 3050 3086 3087 3088 3096 3100 3102 3143 3163 3164 3165 3169 3170 3190 3206 3216 3217 3218 3220 3229 3230 3231 3232 3233 3235 3266 3292 3298 3299 3300 3302 3313 3314 3328 3336 3340 3341 3364 3390 3391 3399 3403 3405 3417 3437 3438 3446 3450 3451 3464 3465 3466 3467 3468 3470 3499 3500 3516 3522 3526 3528 3560 3561 3573 3581 3582 3584 3611 3621 3622 3623 3624 3625 3652 3653 3654 3655 3659 3661 3685 3686 3687 3695 3696 3698 3734 3735 3736 3744 3745 3747 3787 3813 3814 3815 3819 3821 3844 3845 3855 3856 3857 3859 3891 3892 3893 3894 3898 3899 3946 3966 3967 3975 3979 3980 3993 4023 4037 4038 4042 4044 4048 4049 4065 4073 4077 4079 4105 4106 4122 4128 4129 4131 4164 4165 4166 4167 4168 4170 4219 4220 4221 4222 4223 4224 4266 4288 4289 4290 4294 4295 4317 4318 4319 4320 4324 4325 4368 4396 4397 4398 4399 4400 4415 4416 4430 4436 4440 4442 4472 4502 4503 4511 4512 4513 4525 4526 4527 4535 4536 4537 4586 4594 4595 4603 4604 4605 4619 4645 4661 4669 4670 4672 4676 4677 4678 4686 4687 4689 4713 4714 4715 4723 4727 4728 4740 4741 4742 4748 4752 4754 4799 4827 4828 4836 4837 4838 4852 4853 4869 4870 4874 4875 4909 4910 4911 4912 4916 4917 4962 4963 4964 4965 4966 4967 5011 5012 5013 5014 5015 5017 5066 5074 5080 5081 5085 5087 5095 5096 5108 5109 5110 5111 5132 5133 5134 5140 5144 5146 5183 5184 5185 5193 5197 5199 
0 1.0  |txt 0 1 2 10 11 12 57 58 59 67 68 69 114 115 116 124 125 126 171 172 173 181 182 184 232 233 234 235 239 240 293 294 295 303 304 306 346 347 348 349 353 355 399 400 401 402 406 407 454 455 456 457 461 462 509 510 511 512 513 515 562 563 564 572 573 575 621 622 623 631 632 634 672 673 674 682 686 687 729 730 731 732 736 737 784 785 786 787 788 790 837 838 839 840 841 842 884 885 886 894 898 899 945 946 947 948 949 950 1000 1001 1002 1003 1004 1006 1059 1060 1061 1069 1073 1075 1114 1115 1116 1117 1118 1119 1175 1176 1177 1178 1179 1180 1238 1239 1240 1248 1252 1253 1301 1302 1318 1326 1327 1329 1364 1365 1366 1374 1375 1376 1425 1426 1427 1428 1429 1431 1480 1481 1493 1501 1505 1507 1539 1540 1541 1547 1548 1549 1598 1599 1600 1604 1608 1610 1647 1648 1649 1650 1651 1653 1706 1707 1708 1716 1717 1719 1763 1764 1780 1781 1782 1783 1822 1823 1824 1825 1826 1827 1869 1870 1871 1872 1873 1875 1930 1931 1932 1940 1944 1946 1985 1986 1987 1995 1999 2001 2040 2041 2057 2065 2069 2071 2093 2123 2135 2136 2140 2142 2150 2151 2167 2175 2176 2177 2207 2208 2224 2225 2226 2228 2266 2267 2268 2276 2277 2278 2325 2339 2340 2348 2352 2354 2368 2369 2383 2387 2388 2390 2421 2422 2423 2424 2425 2427 2480 2481 2497 2498 2499 2500 2537 2538 2539 2540 2541 2543 2574 2592 2608 2609 2613 2615 2623 2647 2648 2649 2653 2655 2678 2679 2680 2688 2692 2694 2731 2732 2748 2756 2760 2762 2790 2806 2807 2815 2816 2818 2825 2826 2840 2841 2845 2846 2878 2879 2895 2896 2900 2901 2937 2938 2939 2940 2941 2942 2986 3010 3011 3019 3020 3022 3041 3042 3043 3044 3048 3050 3086 3087 3088 3096 3100 3102 3143 3144 3150 3151 3155 3157 3190 3206 3216 3217 3218 3220 3229 3230 3231 3232 3233 3235 3266 3292 3298 3299 3300 3302 3313 3314 3328 3336 3340 3341 3364 3365 3375 3383 3387 3388 3417 3437 3438 3446 3450 3452 3464 3465 3466 3467 3468 3470 3499 3500 3516 3522 3526 3528 3560 3561 3573 3581 3582 3584 3611 3621 3622 3623 3624 3625 3652 3653 3654 3655 3659 3661 3685 3686 3687 3695 3696 3698 3734 3735 3736 3744 3745 3747 3787 3788 3789 3790 3791 3792 3844 3866 3867 3875 3879 3881 3891 3915 3931 3939 3943 3945 3946 3966 3967 3975 3979 3981 3993 4023 4037 4038 4042 4044 4048 4049 4065 4073 4077 4079 4105 4106 4122 4128 4129 4131 4164 4165 4166 4167 4168 4170 4219 4220 4221 4222 4223 4224 4266 4288 4289 4290 4294 4295 4317 4318 4319 4320 4324 4325 4368 4369 4383 4389 4393 4395 4415 4443 4444 4452 4456 4458 4472 4502 4503 4511 4512 4513 4525 4557 4573 4579 4583 4584 4586 4594 4595 4603 4604 4605 4619 4645 4661 4669 4670 4672 4676 4677 4678 4686 4687 4689 4713 4714 4715 4723 4727 4728 4740 4741 4742 4748 4752 4754 4799 4800 4814 4820 4821 4823 4852 4853 4869 4877 4878 4879 4909 4910 4911 4912 4916 4918 4962 4963 4964 4965 4966 4967 5011 5012 5013 5014 5015 5017 5066 5074 5080 5088 5089 5091 5095 5096 5108 5109 5110 5111 5132 5152 5153 5161 5165 5166 5183 5184 5200 5208 5209 5210 
0 1.0  |txt 0 1 2 10 11 12 57 58 59 67 68 69 114 115 116 124 125 126 171 172 173 181 182 184 232 233 234 235 239 240 293 294 295 303 304 306 346 347 348 349 353 355 399 400 401 402 406 407 454 455 456 464 468 470 509 510 511 512 513 515 562 563 564 572 573 575 621 622 623 631 632 634 672 673 674 682 686 687 729 730 731 739 740 741 784 785 786 787 788 790 837 838 839 840 841 842 884 885 886 894 898 899 945 946 956 957 961 963 1000 1001 1015 1023 1027 1028 1059 1060 1061 1069 1073 1075 1114 1115 1131 1132 1136 1138 1175 1176 1192 1193 1197 1199 1238 1239 1240 1248 1252 1253 1301 1302 1318 1326 1327 1329 1364 1365 1366 1374 1375 1376 1425 1426 1427 1428 1429 1431 1480 1481 1493 1501 1505 1507 1539 1540 1554 1555 1559 1561 1598 1599 1600 1604 1608 1610 1647 1648 1649 1650 1651 1653 1706 1707 1708 1716 1717 1719 1763 1764 1780 1781 1782 1783 1822 1823 1824 1825 1826 1827 1869 1870 1886 1887 1891 1893 1930 1931 1932 1940 1944 1946 1985 1986 1987 1988 1992 1993 2040 2041 2042 2050 2051 2053 2093 2123 2135 2136 2140 2142 2150 2151 2167 2175 2176 2177 2207 2208 2224 2225 2226 2228 2266 2298 2314 2315 2317 2318 2325 2339 2340 2348 2352 2353 2368 2369 2383 2387 2388 2389 2421 2422 2423 2431 2435 2437 2480 2481 2497 2498 2499 2500 2537 2538 2539 2540 2541 2543 2574 2592 2608 2609 2613 2615 2623 2647 2648 2649 2653 2655 2678 2679 2680 2681 2685 2686 2731 2732 2733 2741 2742 2744 2790 2806 2807 2815 2816 2818 2825 2855 2869 2870 2878 2879 2895 2896 2900 2901 2937 2938 2939 2947 2948 2950 2986 3010 3011 3019 3020 3022 3041 3042 3043 3044 3045 3046 3086 3087 3088 3096 3100 3102 3143 3144 3150 3151 3152 3154 3190 3206 3216 3217 3218 3220 3229 3230 3231 3232 3233 3235 3266 3292 3298 3299 3300 3302 3313 3314 3328 3336 3340 3341 3364 3365 3375 3383 3384 3385 3417 3437 3438 3439 3440 3441 3464 3465 3466 3467 3468 3470 3499 3500 3516 3522 3526 3528 3560 3561 3573 3581 3582 3583 3611 3621 3637 3638 3642 3644 3652 3653 3654 3655 3659 3661 3685 3686 3687 3695 3696 3698 3734 3735 3736 3737 3738 3739 3787 3788 3789 3790 3791 3792 3844 3866 3867 3875 3879 3881 3891 3892 3893 3894 3898 3899 3946 3966 3967 3968 3972 3974 3993 4023 4037 4038 4042 4044 4048 4049 4065 4073 4077 4079 4105 4106 4122 4128 4129 4131 4164 4165 4166 4167 4168 4170 4219 4220 4221 4222 4223 4224 4266 4288 4289 4290 4294 4295 4317 4318 4319 4320 4321 4323 4368 4369 4383 4389 4393 4395 4415 4443 4444 4445 4446 4448 4472 4502 4503 4511 4512 4513 4525 4526 4527 4528 4529 4531 4586 4594 4595 4603 4607 4609 4619 4645 4661 4669 4670 4672 4676 4677 4678 4686 4687 4689 4713 4714 4715 4716 4717 4718 4740 4741 4742 4748 4752 4754 4799 4800 4814 4820 4821 4823 4852 4853 4869 4870 4874 4875 4909 4910 4911 4912 4916 4918 4962 4963 4964 4965 4966 4967 5011 5012 5013 5014 5015 5017 5066 5074 5080 5081 5082 5083 5095 5096 5108 5116 5117 5118 5132 5133 5134 5140 5144 5145 5183 5184 5185 5193 5194 5195 
