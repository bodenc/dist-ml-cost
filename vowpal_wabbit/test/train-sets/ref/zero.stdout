Derivative 0 detected.
